ci/spack.yaml

```diff
@@ -0,0 +1,19 @@
+# Spack environment file to build NCEPLIBS-grib_util dependencies.
+#
+# Ed Hartnett 12/20/22
+spack:
+  packages:
+    all:
+      compiler:
+      - intel
```

Comment:
Is the plan to compile with GCC, Intel, or both? With one or two other codes that use Spack in their CI we had to do some fiddling to make sure they were reliably using the right compiler.

Reply:
I would like to discuss what the plan is, I'm just groping along here, trying things out. ;-)

Can we build with both GNU and Intel? That is, two completely different builds?

Reply:
Yeah there's a couple ways we can do it. If it's a relatively small stack that you're compiling (which it appears to be), then there's a cool feature I recently discovered that makes it easy to build the whole stack with multiple compilers, so you could do gcc and intel in one go. If you want to have gcc and intel as separate pieces of the workflow, that's also doable though takes a little more work. Wavewatch and UPP are examples of NOAA codes with separate GCC and Intel workflows (though I think this can be implemented with a bit less redundancy in terms of code/config files).